src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -537,3 +541,43 @@ def publish_cluster_nodes(self, node_count: int):
             self.handle_supabase_error()
 
         logging.info(f"cluster nodes: {UPDATE_CLUSTER_NODE_COUNT} => {data}")
+
+    def get_account_resources(self) -> List[AccountResource]:
```

Comment:
I suggest adding parameters to the function like `resource_kind`, and `from_timestamp`. This will make this function reusable for other resources + will allow to fetch only recently updated changes